locale/fa_IR/default.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../dtd/locale.dtd">
+<!--
+  * locale/en_US/default.xml
```

Comment:
Double-check the path names to make sure they're correct -- this one should be `locale/fa_IR/default.xml`, not `locale/en_US/default.xml`.